atoms/default/server/templates/main.html

```diff
@@ -19,7 +19,7 @@
                         <h2 class="newsletter-card__title">The Fiver</h2>
                     
                         <div class="newsletter-card__inner">
-                            <img class="newsletter-card__image" src="<%= path %>/the-fiver-square.png" height="100" width="100">
+                            <img class="newsletter-card__image" src="<%= path %>/the-fiver-square.png" alt="The Fiver newsletter" height="100" width="100">
```

Comment:
```suggestion
                            <img class="newsletter-card__image" src="<%= path %>/the-fiver-square.png" alt="The Fiver newsletter logo" height="100" width="100">
```
Maybe the word logo helps describe the image a bit better? If you agree this would apply to all the alt descriptions